### PR TITLE
fix: Rename UI sub-library.

### PR DESCRIPTION
**Pull request by simensrostad**

### CODE REVIEW

File: src/led/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2019-2020 Nordic Semiconductor ASA
```
Comment (jtguggedal) [inline src/led/CMakeLists.txt]: 2021

File: src/led/led.h [hunk 2]
```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2019-2020 Nordic Semiconductor ASA
```
Comment (jtguggedal) [inline src/led/led.h]: 2021

File: src/led/led.h [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+#define LED_ON_PERIOD_NORMAL 500
+#define LED_OFF_PERIOD_NORMAL 3500
+#define LED_ON_PERIOD_ERROR 500
+#define LED_OFF_PERIOD_ERROR 500
+#define LED_ON_PERIOD_SHORT 350
+#define LED_OFF_PERIOD_LONG 7000
+
+#define LED_MAX 50
+
+#define LED_COLOR_OFF LED_COLOR(0, 0, 0)
```
Comment (jtguggedal) [inline src/led/led.h]: Could we have some indentation to make the definitions more readable?

File: src/led/led_pwm.h [hunk 4]
```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2019-2020 Nordic Semiconductor ASA
```
Comment (jtguggedal) [inline src/led/led_pwm.h]: 2021

**Comment by simensrostad** [comment]
@jtguggedal Comments addressed.

**Review by jtguggedal — APPROVED** [review]

**Pull request merged**